### DISCUSSION TITLE
UIBarButtonItem

### DISCUSSION THREAD

**Discussion by purple-bananas** [sst-inc/help#26] [Q&A - iOS] [answered]
My edit button doesn't appear even after I uncomment the code and I've checked my code multiple times. The app runs but the edit button doesn't show up.

**Comment by theboi** [comment]
Hi, can you send a screenshot of your storyboard and ur zipped up app

check out #23 as well

**Comment by granwyntan** [comment, marked as answer]
Hi @purple-bananas I suspect you have not embedded your initial view controller with a navigation controller.

**Reply by granwyntan** [reply]
If this helps, please mark it as the answer. Thanks!
